### ISSUE TITLE
Print mode

### ISSUE THREAD

**Issue by ralts00** [Js41637/Overwatch-Item-Tracker#209]
Please, it would be great, to be able to have a print mode that is perfect on a letter sheet with everything to mark, currently it is not usable.
![captura de pantalla 2018-04-12 12 04 53](https://user-images.githubusercontent.com/5034252/38692573-c016ddc0-3e49-11e8-8070-21386a3d4f27.png)

Thanks :)

**Comment by Js41637** [comment]
Not a bad idea.

**Comment by ralts00** [comment]
we can build on the great design of  Izzepizze.


![brxdgm9](https://user-images.githubusercontent.com/5034252/38693455-849c029a-3e4c-11e8-84f5-18507422e3da.jpg)

**Comment by Js41637** [comment]
Yeah probably, gotta remember that we can only use CSS to change the layout for print. I cant change the structure and layout of the HTML so there are some limitations but it's definitely possible

**Comment by ralts00** [comment]
I'll try :D
